deploy/kubernetes/elastic-agent-kustomize/default/elastic-agent-managed/base/elastic-agent-managed-daemonset.yaml

```diff
@@ -62,6 +62,10 @@ spec:
               valueFrom:
                 fieldRef:
                   fieldPath: metadata.name
+            # The following ELASTIC_NETINFO:false variable will disable the netinfo.enabled option of add-host-metadata processor. This will remove fields host.ip and host.mac.  
+            # For more info: https://www.elastic.co/guide/en/beats/metricbeat/current/add-host-metadata.html
+            # - name: ELASTIC_NETINFO
+            #   value: "false"
```

Comment:
Shall we make it `true` by default based on the discussions at https://github.com/elastic/beats/pull/36506#issuecomment-1709764228?

Reply:
Setting  default value ELASTIC_NETINFO: false as per discussion here https://github.com/elastic/beats/pull/36506#issuecomment-1709769966